php/Terminus/User.php

```diff
@@ -73,12 +75,21 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
-  public function getId() {
-    return $this->id;
+  public function instruments() {
+    $instruments       = new Instruments(array('user' => $this));
```

Comment:
While I can see sense in moving this constructor, I don't see how anybody can get an error from this. Also, if it's in the constructor and this person never calls instruments(), then we've run API calls nobody needed.